src/components/content/order/Services.tsx

```diff
@@ -42,79 +44,93 @@ function Services(): JSX.Element {
     useEffect(() => {
         setIsServicesLoaded(false);
         const categoryName = location.hash.split('#')[1] as ServiceVo.category;
-        void ServicesAvailableService.getAvailableServicesTree(categoryName).then((rsp) => {
-            const serviceList: { name: string; content: string; icon: string; latestVersion: string }[] = [];
-            if (rsp.length > 0) {
-                rsp.forEach((item) => {
-                    const serviceItem = {
-                        name: item.name || '',
-                        content: item.versions[0].cloudProvider[0].details[0].description,
-                        icon: item.versions[0].cloudProvider[0].details[0].icon,
-                        latestVersion: sortVersion(getVersions(item.versions))[0],
-                    };
-                    serviceList.push(serviceItem);
-                });
-                setServices(serviceList);
+        void ServicesAvailableService.getAvailableServicesTree(categoryName)
```

Comment:
Do we have the same formatter configuration? It seems nothing changed on some lines.

Reply:
@iskey yes the configuration is same. There were changes to if else clauses that's why the formatting changes too.